### PR TITLE
fix: hyphens to en dashes

### PR DESCRIPTION
**Pull request by mattrosno**
Closes https://github.com/carbon-design-system/carbon-platform/issues/1253

Fixes some content bugs found in platform, what are getting remotely rendered from this repo.

We aren't manually fixing straight quotes because we'll take care of that both here and in the platform repo using the smartypants remark plugin.

#### Changelog

**New**

- N/A

**Changed**

- Hyphens to en dashes in the writing style page

**Removed**

- `.vscode/settings.json` should never have been a Git-tracked file, so I removed it

### CODE REVIEW

File: src/pages/guidelines/content/writing-style.mdx [hunk 1]
```diff
@@ -3,15 +3,15 @@ title: Content
 description:
   Writing should always be simple, clear, and easy to understand. Using everyday
   language keeps the tone friendly, human, and inviting. And choose short words
-  over long impressive-sounding words. Put the thesaurus away!
+  over long impressive–sounding words. Put the thesaurus away!
```
Comment (aubrey-oneal) [inline src/pages/guidelines/content/writing-style.mdx:6]: Unrelated to hyphens, but there needs to be a comma between the two coordinate adjectives: long, impressive-sounding...

File: src/pages/guidelines/content/writing-style.mdx [hunk 2]
```diff
@@ -3,15 +3,15 @@ title: Content
...
 tabs: ['Overview', 'Writing style', 'Action labels']
 ---
 
 <PageDescription>
 
 Writing should always be simple, clear, and easy to understand. Using everyday
 language keeps the tone friendly, human, and inviting. And choose short words
-over long impressive-sounding words. Put the thesaurus away!
+over long impressive–sounding words. Put the thesaurus away!
```
Comment (aubrey-oneal) [inline src/pages/guidelines/content/writing-style.mdx:14]: Unrelated to hyphens, but there needs to be a comma between the two coordinate adjectives: long, impressive-sounding...

File: src/pages/guidelines/content/writing-style.mdx (hunk 1 above)
Comment (aubrey-oneal) [inline src/pages/guidelines/content/writing-style.mdx:6]: Unrelated to hyphens, but there needs to be a comma between the two coordinate adjectives: long, impressive-sounding...

**Review by aubrey-oneal — CHANGES_REQUESTED** [review]
Hyphens look good! I'd just suggest adding those two commas.

File: src/pages/guidelines/content/writing-style.mdx (hunk 1 above)
Comment (mattrosno) [reply src/pages/guidelines/content/writing-style.mdx]: Done.

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**